RELEASE_NOTES.md

```diff
@@ -1,6 +1,23 @@
 ### New in 0.45 (not released yet)
```

Comment:
@tabro Could you have a look at this as you would be directly affected. Sort version, asynchronous subscribers are disabled by default and if enabled are executed using the scheduler, i.e., Hangfire in your case.